sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/student-loan-balances/_tests_/student-loan-balances-part-time-integration.scheduler.e2e-spec.ts

```diff
@@ -53,23 +60,20 @@ describe(
 
     beforeEach(async () => {
       jest.clearAllMocks();
+      // Reset all student balances.
+      await db.dataSource.query("TRUNCATE TABLE sims.student_loan_balances");
```

Comment:
this is similar to delete the rows in the table, on every run. I get your point now :)

Reply:
The only difference is that we cannot use `TRUNCATE ` every time but when we need it will be way faster.